Items/08_Tool_sets.txt

```diff
@@ -0,0 +1,125 @@
+-sealing rivets, flash-freezers, etc. Also has a sterilizing unit 
```

Comment:
Top of the document got chopped off

Reply:
@.@

Reply:
Check the re-added first half of the document too please 🐢